### DISCUSSION TITLE
why you stopped publishing nightly build ?

### DISCUSSION THREAD

**Discussion by djouallah** [duckdb/duckdb#11302] [Q&A] [answered]
it seems duckdb stopped publishing nightly build, it is very useful for testing

**Comment by Mause** [comment]
Nightly builds for which platform and language?

**Comment by carlopi** [comment, marked as answer]
I miss some context, but I guess you are referring either to Python or to Java.

Python issue is currently not tracked but it's a known one internally, we are working on bringing them back.
Java is tracked here: https://github.com/duckdb/duckdb/issues/11244, also to be solved soon.

If this is something else, please do raise an issue since I suppose we are not aware of it.

**Comment by djouallah** [comment]
Python

**Reply by carlopi** [reply]
Update: nightlies for Python are back, see https://pypi.org/project/duckdb/0.10.2.dev189/#files or `pip install duckdb==0.10.2.dev189`.

I think this should solve it.

**Reply by djouallah** [reply]
you are the best, but you know that already :)
